package.json

```diff
@@ -17,11 +17,20 @@
     "activationEvents": [
         "onLanguage:robotframework"
     ],
+    "galleryBanner": {
+        "theme": "dark",
+        "color": "#000000"
+    },
     "contributes": {
         "configuration": {
             "title": "Robot Framework Language Server Configuration",
             "type": "object",
             "properties": {
+                "robot.editor.4spacesTab": {
```

Comment:
Regarding the tabs vs spaces, vscode itself should handle that (we shouldn't need to cover this in the extension itself as the vscode default seems to be what we want anyways).

From https://code.visualstudio.com/docs/editor/codebasics:

- Indentation

  VS Code lets you control text indentation and whether you'd like to use spaces or tab stops. By default, VS Code inserts spaces and uses 4 spaces per Tab key. If you'd like to use another default, you can modify the `editor.insertSpaces` and `editor.tabSize` settings.

-- if the user has multiple languages and wants different settings, it should be handled as explained in: https://github.com/Microsoft/vscode/issues/23933

Reply:
This is only half of the truth.
VSCode always (and that can not be disabled) align the spaces to a multiple of the tab size.

That means if i have a keyword that has 8 characters and before are 4 spaces and you press tab, you get 4 spaces.
But if you have a keyword that has 3 characters and 4spaces ahead, you just get 1 space. 


`....Log.`
And
`....Log To Console..`
`....Run Keyword.`

And so on.
This is really annoying!!!
This 4spacesTag adds always exactly 4 spaces. Independent how long the line is, but only if the file is a Robot Framework file and if no text is selected.

I think this is a super cool feature.
We definitely can move this command to the typescript code later in a command listener but for now this is very useful.

And it can be deactivated via a checkbox.